qutebrowser/browser/qtnetworkdownloads.py

```diff
@@ -429,7 +429,8 @@ def get_request(self, request, *, target=None, **kwargs):
                              QNetworkRequest.AlwaysNetwork)
 
         if request.url().scheme().lower() != 'data':
-            suggested_fn = urlutils.filename_from_url(request.url())
+            suggested_fn = (utils.sanitize_filename(title) + ".html" if title
```

Comment:
Oh, good job at finding `utils.sanitize_filename`! I compltely forgot about it 😆